### PR TITLE
Bookmarklet

### PR DESCRIPTION
**Pull request by jcuanm**
When adding images to the Bookmarklet the Description field shows 'none'. The Description field should be blank. Program widget.htm.twig was changed.

### CODE REVIEW

File: src/JDA/CoreBundle/Resources/views/Widget/widget.html.twig [hunk 1]
```diff
@@ -106,7 +106,7 @@
 				<div id="description">
 			
 				<div class="widget-textbox ingest-info">
-				  <textarea id="widget-description" ></textarea>
+				  <textarea id="widget-description" > </textarea>
```
Comment (horak) [inline src/JDA/CoreBundle/Resources/views/Widget/widget.html.twig]: I'm a bit confused, you just added a space? Was this the intended result?

Comment (jcuanm) [inline src/JDA/CoreBundle/Resources/views/Widget/widget.html.twig]: Sorry, got confused with my testing files. I updated it again. "Placeholder" should get rid of "None" in Description field.

**Comment by horak** [comment]
cool!

**Comment by horak** [comment]
This is live!

**Comment by jcuanm** [comment]
I fixed the problem! How can I make a merge request?

**Comment by horak** [comment]
^ everything here is already live as I indicated above (or at least it should be). Can you confirm that?

**Pull request merged**